components/raftstore/src/store/fsm/store.rs

```diff
@@ -115,13 +114,77 @@ pub struct StoreInfo<EK, ER> {
     pub capacity: u64,
 }
 
+#[derive(Default)]
+pub struct StoreMetaRegions {
```

Comment:
How about adding some unit tests to verify the consistency between `regions` and `regions_on_store`?